config/element-api.php

```diff
@@ -834,6 +834,50 @@ function getStatRegions($locale)
     ];
 }
 
+/**
+ * API Endpoint: Data single
+ */
+function getDataPage($locale)
+{
+    normaliseCacheHeaders();
+
+    return [
+        'serializer' => 'jsonApi',
+        'elementType' => Entry::class,
+        'criteria' => [
+            'section' => 'data',
+        ],
+        'one' => true,
+        'transformer' => function (Entry $entry) use ($locale) {
+
+            $stats = [];
+            foreach ($entry->stats as $s) {
+                $statData = [
+                    'title' => $s->statTitle,
+                    'value' => $s->statValue,
+                    'showNumberBeforeTitle' => $s->showNumberBeforeTitle
+                ];
+                if ($s->suffix) {
```

Comment:
With the other API endpoints we've preferred to return `null` rather than no value (makes it easier to see at a glance what the full list of fields are).

Means you can do this as a one-liner if you go that route: 

```php
'suffix' => $s->suffix ?? null
```

Reply:
Done!